src/hotspot/share/gc/shenandoah/c2/shenandoahBarrierSetC2.cpp

```diff
@@ -901,7 +897,7 @@ bool ShenandoahBarrierSetC2::clone_needs_barrier(Node* src, PhaseGVN& gvn) {
       }
     } else {
       return true;
-    }
+        }
```

Comment:
This looks off.

Reply:
Yes, this reverts to "weird' upstream state: 
https://builds.shipilev.net/patch-openjdk-shenandoah/src/hotspot/share/gc/shenandoah/c2/shenandoahBarrierSetC2.cpp.sdiff.html

No reason to have whitespace changes if that is the only change in the hunk.